### PR TITLE
#1705 Add UUID attribute to welcome message

### PR DESCRIPTION
**Pull request by Ryan-Gordon**
couch_server convienently already  has a get_uuid() function located here : https://github.com/apache/couchdb/blob/master/src/couch/src/couch_server.erl#L18

## Overview

Related to issue https://github.com/apache/couchdb/issues/1705
#1705 

From what I could see, the functionality for this was already in place. Just needed to be added to the payload. 
Let me know if I need to change some documentation or modify a test.

## Testing recommendations

The base welcome message located at `GET /` should now have a UUID returned

## Related Issues or Pull Requests

#1705 

## Checklist

- [ ] Code is written and works correctly;
- [ ] Changes are covered by tests;
- [ ] Documentation reflects the changes;

### CODE REVIEW

File: src/chttpd/src/chttpd_misc.erl [hunk 1]
```diff
@@ -50,6 +50,7 @@ handle_welcome_req(#httpd{method='GET'}=Req, WelcomeMessage) ->
         {couchdb, WelcomeMessage},
         {version, list_to_binary(couch_server:get_version())},
         {git_sha, list_to_binary(couch_server:get_git_sha())},
+        {uuid, list_to_binary(couch_server:get_uuid())},
```
Comment (iilyak) [inline src/chttpd/src/chttpd_misc.erl]: `couch_server:get_uuid/0` returns binary (https://github.com/apache/couchdb/blob/master/src/couch/src/couch_server.erl#L68 `?l2b` macro stands for `list to binary`). Attempt to call `list_to_binary` on a binary argument causing `badarg` exception.

**Comment by iilyak** [comment]
The tests for welcome message are here https://github.com/apache/couchdb/blob/master/src/chttpd/test/chttpd_welcome_test.erl

Tests can be run individually using 
```
make eunit apps=chttpd tests=welcome_test_
```

**Comment by iilyak** [comment]
@Ryan-Gordon in addition to new test please squash commits into one, before we merge. Thank you for you contribution. Please don't hesitate to reach out to me if you would have any difficulties with writing the test case.

**Comment by Ryan-Gordon** [comment]
Hello 👍 

I just wrote a test case for this UUID which is very similar to the `should_have_version` test in the same file. If the tests pass happy to have the commits squashed into 1

**Comment by Ryan-Gordon** [comment]
@iilyak So I tried to squash on my own side and what I managed was rolling back 3 commits, committing as one and then a 2nd commit to merge branch

**Comment by wohali** [comment]
@Ryan-Gordon Don't worry about the squashing, I can do that when I merge your changes into `master` with the big "Squash and merge" button. But if you'd like help on git commands, come visit `#couchdb-dev` on Freenode IRC and someone can walk you through it.  Or, [this tutorial](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) might help, it's the one I learned from originally.

Looks like your test passed! Are you ready for me to merge this now?

**Comment by Ryan-Gordon** [comment]
Ahhh yes, the Squash and merge is what im most familiar with ! 
Cool I'll check out the tutorial and maybe join the IRC anyways
Yes all ready to merge !

**Comment by wohali** [comment]
@Ryan-Gordon All done, and thanks again! :confetti_ball:

**Pull request merged**